src/main/java/net/neoforged/neoforge/common/CommonHooks.java

```diff
@@ -1721,4 +1725,54 @@ public static void sendRecipes(ServerPlayer player, Set<RecipeType<?>> recipeTyp
             PacketDistributor.sendToPlayer(player, payload);
         }
     }
+
+    private static final Set<Class<?>> EDA_CHECKED_CLASSES = ConcurrentHashMap.newKeySet(BuiltInRegistries.ENTITY_TYPE.size());
+
+    public static void verifyEntityDataAccessorRegistration(final Class<?> callerClass, final Class<? extends SyncedDataHolder> holderClass) {
```

Comment:
Actually, can we make this method internal? People shouldn't be calling it manually.